### ISSUE TITLE
Font looks really bad in vscode in ubuntu

### ISSUE THREAD

**Issue by santigarcor** [MonoLisaFont/feedback#157]
---

## Background information

* Font edition (i.e. trial, personal, plus, pro): Trial
* Font version (i.e. 1.805): It doesn't say
* The type of the font file you are using (i.e. otf, ttf): otf
* Operating system (i.e. MacOS, Windows): Ubuntu 20.04
* Display resolution (i.e. 3072x1920): 1920x1080 (24'' IPS Led Monitor)

## Issue

I downloaded the trial version to see how it looks on my vscode before buying it and turns out that the font looks really bad in my editor. I tried resetting all the configurations but it still looks weird like low resolution.

I'm really interested in buying the fonts so if you could help me find a solution that would be great

Here you can see it with all my current settings(click on the images to see the issue):
![image](https://user-images.githubusercontent.com/7565417/173209803-cb83b1cf-8961-4fd0-ad2d-75daf1550716.png)
![image](https://user-images.githubusercontent.com/7565417/173209806-ebd0461a-dd53-47d7-a66e-e7b9395bde98.png)

And here you can check it with the recommended config
![image](https://user-images.githubusercontent.com/7565417/173209821-8c7f3476-cd14-4752-94d7-ebfa272aa233.png)
![image](https://user-images.githubusercontent.com/7565417/173209828-aebf53e2-a42f-47c0-97b1-a64632426f91.png)

Here is how it looks with OperatorMono:
![image](https://user-images.githubusercontent.com/7565417/173209925-7229a565-0e97-4a8d-b50f-c0c02e77a629.png)
![image](https://user-images.githubusercontent.com/7565417/173209933-5a98a8b6-377d-497c-9eeb-2a9912607ef3.png)

**Comment by bebraw** [comment]
Out of curiosity, is it any better with ttf? It can be some combination of low resolution and font rendering.

**Comment by santigarcor** [comment]
@bebraw 
With ttf it looks better, but then the ligatures don't work :/
![image](https://user-images.githubusercontent.com/7565417/173236822-60cefc7b-2b8f-4de9-812d-7f7789f419f1.png)
![image](https://user-images.githubusercontent.com/7565417/173236892-dbd10240-e454-4319-84df-48562a96b35a.png)

**Comment by bebraw** [comment]
@santigarcor Which version are you using? The trial version doesn't come with ligatures.

**Comment by santigarcor** [comment]
@bebraw I'm using this version 1.808.

Then why in the first image from the post I see some ligatures?

**Comment by bebraw** [comment]
> Then why in the first image from the post I see some ligatures?

It sounds like a scripting mistake on my side. I've reduced the character range and removed some features. Likely the thing to do is to drop ttf from the trial distribution altogether to limit further.

You would need at least personal version for ligatures to work.

I added a note about ttf on Linux to the FAQ.

As there isn't anything actionable on this issue to fix in the typeface itself, I'm closing the issue.
